src/maestro/cli/streamlit_deploy.py

```diff
@@ -45,7 +45,7 @@ def deploy_agents_workflow_streamlit(agents_file, workflow_file):
         st.session_state.messages = [
             {"role": "assistant", "content": "Welcome to Maestro workflow"}]
 
-    st.image("images/maestro.png", width=200)
+    st.image("https://raw.githubusercontent.com/AI4quantum/maestro/main/images/maestro.png", width=200)
```

Comment:
Is it possible to include the image file in the package?  I may not good to access to github when the maestro command runs.

Reply:
I'll take a look into this, this was more of a fix for when running the demos from a separate directory than the git repo, but you are correct that it's better to include them in the python package and use them directly.